debian/clearwater-cluster-manager.init.d

```diff
@@ -85,10 +85,13 @@ do_start()
     return 2
   fi
 
+  # Make sure to pull in the node_type files before local_config
   local_site_name=site1
   remote_site_name=""
   signaling_namespace=""
   etcd_key=clearwater
+  etcd_cluster_key=""
+  . /usr/share/clearwater/node_type.d/$(ls /usr/share/clearwater/node_type.d | head -n 1)
```

Comment:
This is alphabetical, so `4_bob` is after `10_frank`.  Not an issue for now, but might be surprising in future.